.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+
+    strategy:
+      matrix:
+        go: [ '1.13' ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+
+      - run: gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*") 2>&1
+
+      - run: go build -v .
+
+      - run: go test .
```

Comment:
There are some tests inside the `cancelable` subpackage that won't run if you do it like this, so you should use `go test ./...` instead.